tasktiger/redis_scripts.py

```diff
@@ -469,20 +469,26 @@ def execute_pipeline(self, pipeline, client=None):
         results = redis_scripts.execute_pipeline(p)
         """
 
+        executing_pipeline = None
         try:
+            executing_pipeline = self.redis.pipeline()
+
+            # Load scripts
+            executing_pipeline.script_load(self._execute_pipeline.script)
```

Comment:
Do we need this? Wouldn't Redis properly keep track of this script since it's the one we're directly calling? And `executing_pipeline.execute()` would ensure it's loaded for the first time since it calls `load_scripts()`.